### ISSUE TITLE
Error with Rails 6.0

### ISSUE THREAD

**Issue by cseelus** [glebm/to_spreadsheet#35]
Doesn't work with Rails 6 anymore, unfortunately:

```
Invalid formats: "xlsx", "html"
…/gems/actionview-6.0.0/lib/action_view/lookup_context.rb:288:in `formats='
```

Tested with 1.0.6 and current master from Github.

**Comment by cseelus** [comment]
I've created a bare Rails 6.0 sample app with the current version of to_spreadsheet and Axlsx 3.0.0.pre (also tried 2.1.0.pre with no success), to reproduce the issue: https://github.com/cseelus/to_spreadsheet_rails_6

After a little digging in the Rails method that throws the error, I added `:xlsx` to the `ActionView::Template::Types` in a clean/bare Rails app, which unfortunately didn't do the trick.

At the moment I'm trying out the axlsx-rails example from the docs, but as …

```ruby
respond_to do |format|
  format.html2xlsx
end
```

… expects a `.xslx.axlsx` file, I'm not sure how to_spreadsheet comes into play in this scenario. Using a `index.xlsx.haml` file together with a `.haml`-partial as per the docs, results in a currupted Excel file:

<img width="532" alt="Screen Shot 2019-08-22 at 18 13 17" src="https://user-images.githubusercontent.com/3188392/63531310-85fef680-c508-11e9-9974-8a638150eea3.png">

If you have any suggestion on how to get to_spreadsheet to work with Rails 6.0, I would be more than happy to create a PR to give back at least a  little to this amazing gem.
